utils/test_fv3gfs_dataset.py

```diff
@@ -36,15 +48,18 @@ def __contains__(self, key):
         return key in self.items
 
 
-def test_FV3GFSDataset_init():
-    dataset = FV3GFSDataset(DotDict(TEST_PARAMS), TEST_PATH, True)
-    dataset.in_names == ["UGRD10m", "VGRD10m", "PRMSL", "TCWV"]
-    dataset.out_names == ["UGRD10m", "VGRD10m", "TMP850", "TCWV", "Z500"]
+@pytest.mark.parametrize("params", [TEST_PARAMS, TEST_PARAMS_SPECIFY_BY_NAME])
+def test_FV3GFSDataset_init(params):
+    dataset = FV3GFSDataset(DotDict(params), TEST_PATH, True)
+    assert dataset.in_names == TEST_IN_NAMES
+    assert dataset.out_names == TEST_OUT_NAMES
 
 
 def test_FV3GFSDataset_len():
```

Comment:
```suggestion
def test_FV3GFSDataset_len():
"""Checks that the time variable matches the length of the data."""
```

Reply:
IMO the function name should be sufficient to describe the purpose of the test. Requiring a docstring here feels like a failure of naming. For example, when running `pytest`, it is the function name that will be listed.

The purpose of the test was to ensure that `__len__` method of `FV3GFSDataset` was properly implemented... do you think that was unclear from the function name? The rest is just implementation detail, which, I think, is best understood by actually reading the code versus a comment.